elftools/elf/elffile.py

```diff
@@ -22,6 +26,8 @@
 from .segments import Segment, InterpSegment, NoteSegment
 from ..dwarf.dwarfinfo import DWARFInfo, DebugSectionDescriptor, DwarfConfig
 
+CHUNK_SIZE = resource.getpagesize()
```

Comment:
Since this is used in a single place, I'd prefer it declared next to the usage inside the method